### PR TITLE
Aws ec2 tpb fix

### PR DESCRIPTION
**Pull request by YuvHayun**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: [CIAC-8619](https://jira-hq.paloaltonetworks.local/browse/CIAC-8619)

## Description
A few sentences describing the overall goals of the pull request's commits.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Review by JudahSchwartz — APPROVED** [review]
great work! thanks for going the extra mile!

File: Packs/AWS-EC2/.secrets-ignore [hunk 1]
```diff
@@ -0,0 +1 @@
+::7940
```
[the remark this answers is not in the document]
Comment (YuvHayun) [reply Packs/AWS-EC2/.secrets-ignore:1]: Failed secrets review when I entered role arn argument for a new TPB task

**Comment by YuvHayun** [comment]
@ShahafBenYakir needs force merge on this pr - failing on BC validations (not really a bc, but a fix to sync between the actual inputs/outputs to the ones documented in the yml file.)

**Comment by ShahafBenYakir** [comment]
Great work

**Pull request merged**